docs/user-guide/cli.md

```diff
@@ -195,7 +195,7 @@ stylelint "foo/**/*.css" --config bar/mySpecialConfig.json
 Recursively linting all `.css` files in the `foo` directory using a custom syntax:
 
 ```shell
-stylelint "foo/**/*.css" --customSyntax path/to/my-custom-syntax.js
```

Comment:
This is probably normalized by meow.
see https://github.com/search?q=stylelint+%22--customSyntax%22&type=code

Reply:
Yes, both forms work. But, our help prints only `--custom-syntax`.